python/py-crossbar/Portfile

```diff
@@ -0,0 +1,194 @@
+                    port:py${python.version}-incremental \
+                    port:py${python.version}-jinja2 \
+                    port:py${python.version}-lmdb \
+                    port:py${python.version}-mistune \
+                    port:py${python.version}-netaddr \
+                    port:py${python.version}-passlib \
+                    port:py${python.version}-priority \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-pyubjson \
+                    port:py${python.version}-asn1 \
+                    port:py${python.version}-asn1-modules \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pynacl \
+                    port:py${python.version}-openssl \
+                    port:py${python.version}-pyqrcode \
+                    port:py${python.version}-pytrie \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-sdnotify \
+                    port:py${python.version}-service_identity \
+                    port:py${python.version}-setproctitle \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-treq \
+                    port:py${python.version}-txaio \
+                    port:py${python.version}-txtorcon \
+                    port:py${python.version}-u-msgpack-python \
+                    port:py${python.version}-urllib3 \
+                    port:py${python.version}-watchdog \
+                    port:py${python.version}-werkzeug \
+                    port:py${python.version}-zlmdb \
+                    port:py${python.version}-zopeinterface
+}
+
+subport crossbar {
+    name                crossbar
+    version             ${version}
+    revision            0
+    categories          net
+
+    distfiles
+    distname
+    master_sites
+    patchfiles
+
+    description         Run the crossbar.io WAMP application router as a \
+                        service.
+    long_description    {*}${description}
+
+    depends_build
+    depends_lib         port:py${python.version}-crossbar
+
+    set cb_config_path  ${prefix}/etc/${name}
+    set cb_data_path    ${prefix}/var/db/${name}
+    set cb_log_path     ${prefix}/var/log/${name}
+    set cb_share_path   ${prefix}/share/${name}
+
+    set cb_plist_path   ${prefix}/etc/LaunchDaemons/org.macports.${name}
+
+    set cb_config_file  ${cb_config_path}/config.json
+    set cb_config_src   ${cb_share_path}/config.json.example
+    set cb_log_file     ${cb_log_path}/crossbar.log
+
+    set cb_plist        org.macports.${name}.plist
+
+    set cb_bin          ${prefix}/bin/crossbar-${python.branch}
+
+    set cb_user         _crossbar
+
+    add_users           ${cb_user} \
+                        group=${cb_user} \
+                        realname=Crossbar.io
+
+    pre-fetch {}
+    fetch {}
+    pre-build {}
+    build {}
+    pre-checksum {}
+    checksum {}
```

Comment:
All you need here to prevent the build phase from running is:
```
build {}
```
The other lines can be removed. `pre-` and `post-` blocks are cumulative, so you're merely defining additional empty procedures here that will be called in addition to any other existing `pre-` and `post-` blocks. And the fetch and checksum phases won't do anything anyway since you've cleared `distfiles` above.

Reply:
Change made.